scripts/getPVAliases.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/bash
+
+# define usage
+usage()
+{
+cat << EOF
+usage: gatherPVAliases [-h] [-d] patt hutch
+
+gathers all record <-> alias associations from a child's ioc.cfg, st.cmd, and
+parent ioc.cfg.
+
+positional arguments:
+  patt
+  hutch
+
+options:
+  -h, --help     show this help message and exit
+  -d, --dry_run  Forces a dry run for the script. No files are saved.       
+
+EOF
+}
+
+# catch improper number of args or asking for help
+
+if [[ ("$#" -lt 2) || ("$1" == "-h") || ("$1" == "--help") ]]; then
+    usage
+    exit 0
+fi
+
+# source into pcds_conda if not currently active
+
+if [[ ($CONDA_SHLVL == 1) ]];then
+    if [[ "$(echo $CONDA_PREFIX | grep 'pcds-')" ]];then
+        :
+    else
+        source pcds_conda
+    fi
+else
+    source pcds_conda
+fi
+
+# execute python script
+python "$ENG_TOOLS_SCRIPTS/getPVAliases.py" $@
```

Comment:
Feel free to also try something like this if sourcing pcds_conda is ever slow. It's likely that your script doesn't need the full environment sourced, and rather just needs the executable.

This also means you'd be able to do all the arg handling in the python script (if you removed source pcds_conda)
```suggestion
/cds/group/pcds/pyps/conda/py39/envs/pcds-5.8.2/bin/python "$ENG_TOOLS_SCRIPTS/getPVAliases.py" $@
```

Reply:
I very much prefer this, and quite honestly, I'd rather do that and reduce the whole `getPVAliases.sh` to this, since all other arg handling is done by argparser and is now very easy to maintain.